Source/JavaScriptCore/API/tests/testapi.c

```diff
@@ -1195,13 +1195,13 @@ static void checkJSStringOOBUTF8(void)
     const size_t sourceCStringSize = 200;
     const size_t outCStringSize = 10;
 
-    char sourceCString[sourceCStringSize];
-    memset(sourceCString, 0, sizeof(sourceCString));
+    char *sourceCString = (char*)malloc(sourceCStringSize);
+    memset(sourceCString, 0, sourceCStringSize);
     for (size_t i = 0; i < sourceCStringSize - 1; ++i)
         sourceCString[i] = '0' + (i%10);
 
-    char outCString[outCStringSize + sourceCStringSize];
-    memset(outCString, 0x13, sizeof(outCString));
+    char *outCString = (char*)malloc(outCStringSize + sourceCStringSize);
+    memset(outCString, 0x13, sizeof(outCStringSize + sourceCStringSize));
```

Comment:
Should this just be `memset(outCString, 0x13, outCStringSize + sourceCStringSize)`?

Reply:
Yep!  Copy-pasta error.

Reply:
Good catch!